lib/suse-cloud-upgrade-4-to-5-post

```diff
@@ -187,6 +186,20 @@ if [ -f "/opt/dell/crowbar_framework/barclamps/pacemaker.yml" ]; then
   fi
 fi
 
+### Updating default session timeout in nova_dashboard
+proposals=$(crowbar nova_dashboard proposal list)
+if [ "$proposals" != "No current proposals" ]; then
+  echo_summary "Setting default Horizon session timeout."
+  work_dir="${UPGRADE_DATA_DIR}/nova_dashboard"
+  mkdir -p $work_dir
+
+  for proposal in $proposals; do
+    proposal_json="$work_dir/${proposal}.json"
+    crowbar nova_dashboard proposal show $proposal > "$proposal_json"
+    $json_edit $proposal_json -a "attributes.nova_dashboard.session_timeout" -v "240" 
```

Comment:
Might be worth having a comment to explain why 240; I know that it's 14400 / 60 where 14400 is the keystone setting (and here we're using minutes, not seconds). But somebody who doesn't know that would need an explanation.